tests/Integration/Behaviour/Features/Scenario/CatalogPriceRule/catalog_price_rules_listing.feature

```diff
@@ -1,7 +1,7 @@
-#./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags catalog-price-rule-listing
+#./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s catalog_price_rule --tags catalog-price-rule-listing
```

Comment:
these tests failingl. The tags were incorrect previously, but I also tried testing it on develop and got same errors, so Im not sure if it was never working or whats the problem here.
![image](https://user-images.githubusercontent.com/31609858/224973986-3953ac9b-3b8e-48c7-bf08-0c1acbd8fa4b.png)

Reply:
oh it passed in git actions eventually, so probably all good :man_shrugging: